numeric_ranges_algorithms/numeric_ranges_algorithms.md

```diff
@@ -717,7 +717,7 @@ range-as-output overloads to coexist, so we follow [@P3179R8] by not proposing i
 Regarding (2), we make the parallel algorithms proposed here take sized random access ranges, as [@P3179R8] does.
 For consistency, we also propose that the output ranges be sized. As a result, any parallel algorithms with an output range
 need to return both an iterator to one past the last element of the output, and an iterator to one past the last element of
-the input. This tells callers whether there was enough room in the output, and if not, where to start when processing the
+the input. This tells callers whether there was enough room in the output, and if not, where to start while processing the

```

Comment:
"While" suggests a continuing process.  "When" is the time at which the rest of the input will be processed.  The latter is what we want to imply.